### ISSUE TITLE
GUI: Be nice if Esc closed the About CaPTk window. Or ctrl-W.

### ISSUE THREAD

**Issue by eac-upenn** [CBICA/CaPTk#719]

**Comment by ashishsingh18** [comment]
This one was such a pain...

**Comment by eac-upenn** [comment]
> This one was such a pain...

I only said it would be *nice*. Not essential :-) Anyway, speaking as someone who knows nothing about this, don't you have some window-management library that manages midgy things like this for you? :-)

**Comment by ashishsingh18** [comment]
> 
> 
> > This one was such a pain...
> 
> I only said it would be _nice_. Not essential :-) Anyway, speaking as someone who knows nothing about this, don't you have some window-management library that manages midgy things like this for you? :-)

We do use a library for user interface. I tried more than 5 different ways ( all of which should have worked ) to get the Esc behavior to work but none of them did. There was a problem with 'key press' events. The page was not getting any key press events. I believe they were not being propagated properly. The weird thing was that this has been working for all other pages except the 'About' page.

I found a workaround and this has been fixed now. So we are good. What should have been a very easy fix ended up taking a lot of time...
